src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -48,6 +48,10 @@
   <module name="org.openhab.tools.analysis.checkstyle.OverridingParentPomConfigurationCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
+      <property name="severity" value="warning" />
```

Comment:
By the way, I checked all of these bundles. I don't think any of them really needs the `Require-Bundle` header. It can either be removed, or the required package could be moved to `Import-Package` header. @kaikreuzer, if you want I can make a PR about that in `openhab2-addons` repo.